src/main/java/io/supertokens/webserver/api/oauth/OAuthTokenAPI.java

```diff
@@ -88,19 +88,17 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
                         AppIdentifier appIdentifier = getAppIdentifier(req);
                         Storage storage = enforcePublicTenantAndGetPublicTenantStorage(req);
 
-                        JsonObject accessTokenUpdate = InputParser.parseJsonObjectOrThrowError(bodyFromSDK, "access_token", true);
-                        JsonObject idTokenUpdate = InputParser.parseJsonObjectOrThrowError(bodyFromSDK, "id_token", true);
+                        JsonObject accessTokenUpdate = InputParser.parseJsonObjectOrThrowError(input, "access_token", true);
+                        JsonObject idTokenUpdate = InputParser.parseJsonObjectOrThrowError(input, "id_token", true);
```

Comment:
Please move parsing/getting these up: we want to trigger any input validation errors before making calls to hydra.

Reply:
done